src/app/domain/schema-management/components/manual-verification-wizard.component.html

```
@@ -0,0 +1,122 @@
+          <p class="text-sm text-green-700 dark:text-green-300">
+            <strong>Verification Complete!</strong> You have successfully verified the data and obtained the audit hash. You may now close this wizard.
+          </p>
+        </div>
+      </div>
+    }
+  </div>
+
+  <!-- Footer Actions -->
+  <div class="px-6 py-4 border-t border-border-subtle bg-gray-50/80 dark:bg-slate-900/50 flex justify-between items-center">
+    <div>
+      <button (click)="close(true)" class="text-sm text-gray-500 hover:text-gray-700 dark:text-slate-400 dark:hover:text-slate-200 underline focus:outline-none">
```

HTMLHint (1.9.2)

[warning] The type attribute must be present on <button> elements.

(button-type-require)